client/src/main/kotlin/no/nav/su/se/bakover/client/dokarkiv/Journalpost.kt

```diff
@@ -257,8 +261,6 @@ data class Fagsak(
 
 data class JournalpostDokument(
     val tittel: String,
-    val dokumentKategori: DokumentKategori,
-    val brevkode: String = "XX.YY-ZZ",
```

Comment:
Har denne noen betydning eller er det bare en todo verdi av noe slag? 

Fra docs: 
Kode som sier noe om dokumentets innhold og oppbygning. Brevkode bør settes for alle journalposttyper, og brukes blant annet for statistikk.
For inngående dokumenter kan brevkoden for eksempel være en NAV-skjemaID f.eks. "NAV 14-05.09" eller en SED-id.
For utgående dokumenter og notater er det systemet som produserer dokumentet som bestemmer hva brevkoden skal være. Om fagsystemet har "malkoder" kan man gjerne bruke disse som brevkode.

Reply:
Ja, men det jeg mener, har dette egentlig noe signifikant mening, altså noe vi burde forholde oss til? eller kan vi bare ikke ha den?

Reply:
Når man har over 1500 endringslinjer, mener jeg slike ting/diskusjoner er feature creep og bør ut av PRen og inn i et kort også kan man gjøre den analysen på siden.